### DISCUSSION TITLE
[Support]: Is there a way to go from event to recording?

### DISCUSSION THREAD

**Discussion by agrieco** [blakeblackshear/frigate#11120] [General Support] [answered]
### Describe the problem you are having

I've got an number of situations where I get an event recording that I want to go from there to the recording itself to see additional time around the event.  Is there some easy way to click on something that takes me from the event recording to at least the full recording of the hour that contains that event?  Bonus points to seek to the event start time itself. 

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
This is a UI question.
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

NA

**Comment by NickM-27** [comment, marked as answer]
Currently no, but this is already implemented in the new ui

**Reply by agrieco** [reply]
thank you.
